src/bitcoin/mod.rs

```diff
@@ -107,25 +107,28 @@ pub struct Config {
     /// If a signer does not submit signatures for this many consecutive
     /// checkpoints, they are considered offline and are removed from the
     /// signatory set (jailed) and slashed.
-    #[orga(version(V1, V2, V3, V4))]
+    #[orga(version(V1, V2, V3, V4, V5))]
     pub max_offline_checkpoints: u32,
     /// The minimum number of confirmations a checkpoint must have on the
     /// Bitcoin network before it is considered confirmed. Note that in the
     /// current implementation, the actual number of confirmations required is
     /// `min_checkpoint_confirmations + 1`.
-    #[orga(version(V2, V3, V4))]
+    #[orga(version(V2, V3, V4, V5))]
     pub min_checkpoint_confirmations: u32,
     /// The maximum amount of BTC that can be held in the network, in satoshis.
-    #[orga(version(V2, V3, V4))]
+    #[orga(version(V2, V3, V4, V5))]
     pub capacity_limit: u64,
 
-    #[orga(version(V4))]
+    #[orga(version(V4, V5))]
     pub max_deposit_age: u64,
 
-    #[orga(version(V4))]
+    #[orga(version(V4, V5))]
     pub fee_pool_target_balance: u64,
-    #[orga(version(V4))]
+    #[orga(version(V4, V5))]
     pub fee_pool_reward_split: (u64, u64),
+
+    #[orga(version(V5))]
```

Comment:
Also though - it does seem surprising that a relayer change involves a change to on-chain state, since this buffer doesn't affect consensus. Could we avoid that by using a constant e.g. the buffer period is 10% of `max_deposit_age`?